cmd/build.go

```diff
@@ -40,25 +39,17 @@ See our documentation to learn more:
 https://docs.snyk.io/products/snyk-infrastructure-as-code/custom-rules/getting-started-with-the-sdk/bundling-rules
 `,
 	SilenceUsage: true,
-	Args: func(cmd *cobra.Command, args []string) error {
-		if len(args) > 1 {
-			return errors.New("Too many paths provided")
-		}
-		return nil
-	},
 	PostRun: func(cmd *cobra.Command, args []string) {
-		var path string
 		if len(args) == 0 {
```

Comment:
It'd seem that in case no paths are provided and the default paths are to be used, the result from the `CheckIfRunningInRootDirectory` would always be `true`.

Perhaps the execution of this function would simply be unnecessary in this case.

If so, the code could be refactored a bit to look like:
```
PostRun: func(cmd *cobra.Command, args []string) {
		if len(args) == 0 {
			return
		}
		util.CheckIfRunningInRootDirectory(args)
	},
```

Reply:
Just realised also that even if someone provides the path, that doesn't mean it actually exists. So in the next commit I'll fix that as well